nginx/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Nginx image Dockerfile serving learning locker application
+FROM nginx:1.15
+
+COPY nginx.conf.template /etc/nginx/conf.d/default.conf.template
+
+COPY entrypoint.sh /entrypoint.sh
+
+ENTRYPOINT ["/entrypoint.sh"]
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
I don't think we need this image at all. As you suggested IRL, we can use the official nginx image combined with `envsubst`.